### ISSUE TITLE
Iperf3 plugin doesn't detect failures early enough

### ISSUE THREAD

**Issue by mfeit-internet2** [perfsonar/pscheduler#1270]
The iperf3 plugin should look for `error` in the output before attempting to parse it as a valid result.

```
Error:
  iperf3 output has neither 'sum_sent' nor 'sum' field, and one of them is required

Diagnostics:
  Participant 0:
  numactl -N 0 /usr/bin/iperf3 -p 5201 -B 131.215.113.136 -c 208.69.128.92 -t 5 --json --rsa-public-key-path /run/pscheduler-server/runner/tmp/tmpe3wcueoa/tmp8i2x4ie8/public-key --username gYPQGeD9oKft7u7hCUl2

  {
  	"start":	{
  		"connected":	[],
  		"version":	"iperf 3.11",
  		"system_info":	"Linux perfsonar 3.10.0-1160.59.1.el7.x86_64 #1 SMP Tue Feb 22 10:57:56 CST 2022 x86_64",
  		"timestamp":	{
  			"time":	"Tue, 08 Mar 2022 15:37:05 GMT",
  			"timesecs":	1646753825
  		},
  		"connecting_to":	{
  			"host":	"208.69.128.92",
  			"port":	5201
  		},
  		"cookie":	"a2uigi7hhgd5lxos3nrh3nl6gsxy4zihfjld",
  		"tcp_mss_default":	8948,
  		"target_bitrate":	0
  	},
  	"intervals":	[],
  	"end":	{
  	},
  	"error":	"test authorization failed"
  }
```
